### PR TITLE
Add a client example written in React

### PR DESCRIPTION
**Pull request by Jasopaum**

### CODE REVIEW

**Review by elsiehoffet-94 — APPROVED** [review]
👏 Cool, thanks!

File: example-client-react/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    "@types/node": "^12.0.0",
+    "@types/react": "^16.9.0",
+    "@types/react-dom": "^16.9.0",
+    "axios": "^0.20.0",
+    "client-oauth2": "^4.3.3",
+    "query-string": "^6.13.5",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-scripts": "3.4.3",
+    "typescript": "~3.7.2"
```
Comment (simonvadee) [inline example-client-react/package.json:20]: please split dependencies and devDependencies

File: example-client-react/src/components/home/index.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+          </header>
+        </div>
+      );
+    }
+  }
+
+  return (
+    <div className="App">
+      <header className="App-header">
+        <p>Connected with token starting with {accessToken.substring(0, 15)}</p>
```
Comment (simonvadee) [inline example-client-react/src/components/home/index.tsx:80]: I think you can display the whole token here ?

**Review by simonvadee — CHANGES_REQUESTED** [review]

File: example-client-react/src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+import axios from "axios";
+
+import Home from "./components/home";
+
+import { refreshToken, removeTokens } from "./oauth/tokenManager";
+import { ACCESS_TOKEN_STORAGE_KEY, TOKEN_URL } from "./constants";
+
+const redirectToLogin = () => {
+  removeTokens();
+};
```
[the remark this answers is not in the document]
Comment (Jasopaum) [reply example-client-react/src/App.tsx:12]: It's not, indeed 👍

**Review by simonvadee — APPROVED** [review]

**Pull request merged**